static/js/script.js

```diff
@@ -1,4 +1,18 @@
- //modal close button
+function addAnchorTags() {
```

Comment:
Why can't you just use the default anchorjs call? This looks super complicated.

Reply:
Sorry I missed that. I think we should talk about this as a SIG. My personal gut-reaction opinion is that different icons will create visual inconsistency that users might not be able to easily discern the purpose of, since to notice what level a heading is at, you need to scroll back up to its parent, etc. When people grab the link, what they want is to be able to send someone (or their own browser via a bookmark) exactly there. They don't necessarily care what level it is at. I might totally be off base, though. It is an interesting idea.

Reply:
using the same icons/symbols for all heading levels seems OK.

Reply:
@mistyhacks I agree that a single icon seems like a good default for now (I've chosen **#** as that seems to be standard in the industry). I've already made that change, which you can have a look at in the preview build. We can leave open the option of a more fine-grained system down the line.

Reply:
Where are you seeing `#` being used? I have seen https://www.compart.com/en/unicode/U+1F517 (the default for AnchorJS) used quite heavily in multiple docsets. That said, it isn't a deal breaker for me.

Reply:
@mistyhacks Okay, that works for me!

Reply:
One more nit: When I look at https://deploy-preview-10155--kubernetes-io-master-staging.netlify.com//docs/setup/independent/kubelet-integration/#workflow-when-using-kubeadm-join, the <code> in the title should really be rendering as mono text, not bolder than the surrounding text. It's getting the bold from `#docsContent code`. The easiest thing to do is remove the `font-family: inherit` from the following selector, and it looks better to my eyes to set the font-size just slightly smaller too:

```
#docsContent h1 code, #docsContent h2 code, #docsContent h3 code, #docsContent h4 code, #docsContent h5 code, #docsContent h6 code {
    /* font-family: inherit; */
    font-size: inherit;
    background-color: transparent;
    font-size: 85%;
}
```